chapter-2.qmd

```diff
@@ -0,0 +1,261 @@
+---
+title: "Data visualization"
+---
+
+## Notes
+
+`alpha` aesthetic to add transparency to the filled density curves.
+
+This aesthetic takes values between 0 (completely transparent) and 1 (completely opaque)
+
+is to split your plot into **facets**, subplots that each display one subset of the data.
```

Comment:
This seems like an incomplete sentence?

Reply:
Since this is just in notes which won't be part of the final product, I'll resolve this comment without addressing it.